wgpu-core/src/binding_model.rs

```diff
@@ -50,6 +50,8 @@ pub enum CreateBindGroupLayoutError {
     TooManyBindings(BindingTypeMaxCountError),
     #[error("Binding index {binding} is greater than the maximum index {maximum}")]
     InvalidBindingIndex { binding: u32, maximum: u32 },
+    #[error("Invalid visibility {0:?}")]
```

Comment:
This will likely print nothing if made up of fully invalid bits. We should probably print the raw u32 or something.

Reply:
bitflags handles it surprisingly well, for example:

```rust
let flags = unsafe { wgpu::BufferUsages::from_bits_unchecked(0b1100000001) };
println!("{:?}", flags);
```

prints `MAP_READ | INDIRECT | 0x200`